test/test_autograd.py

```diff
@@ -4499,6 +4502,105 @@ def fn(*inputs):
 
             setattr(TestAutogradDeviceType, test_name, do_test)
 
+class TestAutogradComplex(TestCase):
+    # remove this test after gradcheck support is added for non-holomoprphic functions
+    def test_real(self):
```

Comment:
Can these tests also check a sum over a couple dimensions to ensure the gradient is not always uniform?